platforms/android/com/crittercism/plugin/CDVCrittercism.java

```diff
@@ -98,6 +104,14 @@ public boolean execute(String action, final JSONArray args,
         }
     }
 
+    private boolean executeInit(String action, final JSONArray args,
+                               CallbackContext callbackContext) throws JSONException {
+        final String appID = args.getString(0);
```

Comment:
We should probably do some validation here and check if "args" has length >= 1.
A customer could potentially leave out the parameters on the javascript side.